pkg/commands/compute/manifest/manifest.go

```diff
@@ -225,6 +226,12 @@ type Backend struct {
 	OverrideHost string `toml:"override_host,omitempty"`
 }
 
+// Dictionary represents a dictionary to be mocked by the local testing server.
+type Dictionary struct {
+	File   string `toml:"file"`
+	Format string `toml:"format"`
```

Comment:
Does `format` need `omitempty`? 

By this I mean: is it a required property, or will the file format be inferred (or a default used) by Viceroy?

Reply:
It is required.

Reply:
OK cool 👍🏻